references/video_classification/README.md

```diff
@@ -0,0 +1,33 @@
+# Video Classification
+
+TODO: Add some info about the context, dataset we use etc
```

Comment:
@bjuncek can you enhance this with information that you used to train on the datasets? We could potentially refer to submitit